RecoEgamma/EgammaElectronAlgos/interface/ElectronSeedGenerator.h

```diff
@@ -38,9 +37,11 @@
 #include "FWCore/Framework/interface/ESHandle.h"
 #include "FWCore/Framework/interface/EDConsumerBase.h"
 
+#include "TrackingTools/KalmanUpdators/interface/KFUpdator.h"
+
 class PropagatorWithMaterial;
 class KFUpdator;
-class PixelHitMatcher;
+class PixelHitMatcher_;
```

Comment:
is this used in this file?

Reply:
No, that's an artefact of find-replace. Actually all the forward declarations can be removed.